config/settings.global.json

```
@@ -2,6 +2,7 @@
 	"external_tools":
 	{
 		"eagle": "REPLACE WITH PATH TO EAGLE",
-		"gpsim": "REPLACE WITH PATH TO GPSIM IF YOU WANT TO SIMULATE PIC12 AND PIC16 (REQUIRED FOR UNIT TESTS)"
+		"gpsim": "REPLACE WITH PATH TO GPSIM IF YOU WANT TO SIMULATE PIC12 AND PIC16 (REQUIRED FOR UNIT TESTS)",
+		"sim30": "REPLACE WITH PATH TO SIM30 IF YOU WANT TO SIMULATE PIC24 (REQUIRED FOR UNIT TESTS)"
```

I agree with this.  How would you feel about having a strategy of first looking for an environment variable if one is set and second checking if the executable is in the PATH?  If neither of the two works then we could raise an exception.  That would allow for vagrant to set up the environment variables correctly but also mean that people wouldn't have to pollute their environment variables with lots of random definitations if they didn't want to use vagrant